src/Backend/Form/Type/ImageType.php

```diff
@@ -0,0 +1,279 @@
+                        }
+
+                        $imageClass = $options['image_class'];
+
+                        if (!$image instanceof AbstractImage) {
+                            // your editor might say the file has protected but it isn't in this case
+                            $image = $imageClass::fromUploadedFile($image->file);
+                        }
+
+                        $this->nextField();
+
+                        if ($this->isNew()) {
+                            if ($this->getRemoveField() !== null && $this->getRemoveField()->getData()) {
+                                return $imageClass::fromUploadedFile();
+                            }
+
+                            return clone $image;
+                        }
+
+                        if ($this->getRemoveField() !== null && $this->getRemoveField()->getData()) {
+                            $image->markForDeletion();
+
+                            return clone $image;
+                        }
+
+                        // return a clone to make sure that doctrine will do the lifecycle callbacks
+                        return clone $image;
+                    }
+                )
+            );
+    }
+
+    /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setRequired(
+            [
+                'image_class',
+                'show_preview',
+                'show_remove_image',
+                'remove_image_label',
+                'required_image_error',
+                'preview_image_directory',
+                'accept',
+            ]
+        );
+
+        $resolver->setDefaults(
+            [
+                'data_class' => AbstractImage::class,
+                'empty_data' => function () {
+                    $emptyData = new StdClass();
+                    $emptyData->file = null;
+
+                    return $emptyData;
+                },
+                'compound' => true,
+                'preview_class' => 'img-thumbnail img-responsive',
+                'show_preview' => true,
+                'show_remove_image' => true,
+                'required_image_error' => 'err.FieldIsRequired',
+                'remove_image_label' => 'lbl.Delete',
+                'preview_image_directory' => 'source',
+                'accept' => 'image/*',
+                'constraints' => [new Valid()],
+                'error_bubbling' => false,
+                'help_text_label' => 'msg.HelpImageFieldWithMaxFileSize',
+                'help_text_argument' => $this->getUploadMaxFileSize(),
+            ]
+        );
+    }
+
+    /**
+     * @return string
+     */
+    public function getBlockPrefix()
+    {
+        return 'image';
+    }
+
+    /**
+     * @return string
+     */
+    public function getParent()
+    {
+        if (!$this instanceof self) {
+            return 'image';
```

Comment:
fixed in d68ce22d7
and no :) it has to be file because we don't want the custom stuff of fork_file to interfere with fork_image :) file is the default of symfony, there is non for image that's why it originally wasn't prefixed with fork